rust-runtime/aws-smithy-http-server-python/src/error.rs

```diff
@@ -13,7 +13,7 @@ use aws_smithy_http_server::{
     response::IntoResponse,
 };
 use aws_smithy_types::date_time::{ConversionError, DateTimeParseError};
-use pyo3::{create_exception, exceptions::PyException as BasePyException, prelude::*, PyErr};
```

Comment:
Just curious, was this an unused import?

Reply:
Yes it was, `PyErr` was part of `prelude::*`